test/test_xpu.py

```diff
@@ -229,6 +230,13 @@ def test_events(self):
         stream.record_event(event)
         event.synchronize()
         self.assertTrue(event.query())
+        start_event = torch.xpu.Event(enable_timing=True)
+        end_event = torch.xpu.Event(enable_timing=True)
+        stream.record_event(start_event)
+        time.sleep(0.1)
```

Comment:
Here we should submit some kernels, instead of host sleep?

Reply:
Here intended to use host sleep in case the over-optimization that the driver uses the last event to measure the time(will be zero).